src/app/features/course/course.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, Input } from '@angular/core';
-import { Course } from '../courses/courses.model';
+import { Course } from '../courses/courses-model';
 
 @Component({
```

Comment:
Do we still need this method `joinAuthors()`?

Reply:
deteted